### DISCUSSION TITLE
Chat analyzing and navigation

### DISCUSSION THREAD

**Discussion by patrickdalla** [sepinf-inc/IPED#2331] [General]
I'm analyzing a big chat with images. Eventually, when a relevant image is found, I need to view it to copy it and its metadata information to transcribe to the report. I do it checking the image in chat and searching the checked item in the references tab. 
But after that, I don't know how to return to the same position I was in chat. Is it possible in current implementation version? Maybe there is a way, and I only need to know how to do it. It would save some navigation time.

**Comment by wladimirleite** [comment]
Many people request something like you described. I implemented a prototype of an auxiliary viewer, so you don't need to leave the chat. But we are still discussing how is the best way.
#2206
